src/scanner.zig

```diff
@@ -484,11 +493,11 @@ const Interface = struct {
 
         try writer.print(
             \\ = struct {{
-            \\ extern const {s}_interface: common.Interface;
+            \\ extern const {[in]}_interface: common.Interface;
             \\ pub fn getInterface() callconv(.Inline) *const common.Interface {{
-            \\  return &{s}_interface;
+            \\  return &{[in]}_interface;
             \\ }}
-        , .{ interface.name, interface.name });
+        , .{ .in = fmtId(interface.name) });
```

Comment:
this shouldn't use `fmtId()` as the produced code would be invalid if interface.name was an identifier in the current scope:
```zig
extern const @"foobar"_interface
```